ProfilerGUI/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ProfilerGUI")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("ProfilerGUI")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```

Comment:
unfortunately it can't be put in a central location except with some ugly VS tricks which I don't like: https://stackoverflow.com/questions/6771694/shared-assemblyinfo-for-uniform-versioning-across-the-solution

Reply:
For GUI/UploadDaemon: Why can't we go with one `AssemblyInfo.cs` file thats linked to both projects? That's what we do in the Teamscale VS plugin and it seems to work just fine there.

To make it work for the Profiler C++ project as well, I'd imagine we could place it in some properties file and read that when setting up the environment. I'm not sure that's technically feasible though...